required/latex-lab/latex-lab-sec.dtx

```diff
@@ -637,11 +639,20 @@
 \def\hyper@nopatch@sectioning{}
 %    \end{macrocode}
 %
-% \begin{macro}{\@hyp@section@target@nnn}
-% A simple internal command. There is no need for something public,
+% \begin{macro}{\@hyp@section@target@nnn,\@hyp@section@target@n}
+% Two simple internal commands. There is no need for something public,
 % as packages defining their own version of \cs{@startsection} will 
 % probably need something slightly different based on \cs{MakeLinkTarget}. 
+% \changes{v0.84i}{2025/02/13}{Use only an external wrapper with one argument, 
+% to allow to hide the argument from \cs{MakeUppercase},
+% tagging-project issue \#787}
 %    \begin{macrocode}
+\cs_new_protected:Npn \@hyp@section@target@n #1 %
```

Comment:
can the argument contains \par (in optarg)?

Reply:
No. This is a prefix for a destination name (so that you get xyz.1.2. instead of say section.1.2 in the destination).